docs/docs/notification-center/headless/headless.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 8
+---
+
+# Headless Notification Center
```

Comment:
@BiswaViraj do we have some example on how to use it with the socket events of notification count change and etc...?